From a812d071294832e47a3edbadba250616626b95b7 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <bochecha@fedoraproject.org>
Date: Mon, 24 Jan 2011 16:07:23 +0800
Subject: [PATCH] Don't ask questions at build time.

Builds must not be interactive in Fedora. Let's just use the defaults
and avoid questions.
---
 Makefile.PL |    4 +---
 1 files changed, 1 insertions(+), 3 deletions(-)

diff --git a/Makefile.PL b/Makefile.PL
index 2a31180..28d8ddc 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -27,9 +27,7 @@ and hope for the best. The defaults should usually do.
 
 EOF
 
-if (prompt ("Skip further questions and use defaults (y/n)?", "y") =~ /[yY]/) {
-   $ENV{PERL_MM_USE_DEFAULT} = 1;
-}
+$ENV{PERL_MM_USE_DEFAULT} = 1;
 
 print <<EOF;
 
-- 
1.7.3.4

